From: Michael Biebl <biebl@debian.org>
Date: Thu, 2 Jun 2016 17:58:35 +0200
Subject: systemd: Use RemainAfterExit=yes for
 NetworkManager-wait-online.service

RemainAfterExit=yes is typically used for Type=oneshot services.
systemd-networkd-wait-online.service uses it, so we should as well.

See
https://www.freedesktop.org/software/systemd/man/systemd.service.html#RemainAfterExit=
https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=

https://bugzilla.gnome.org/show_bug.cgi?id=767170
---
 data/NetworkManager-wait-online-systemd-pre200.service.in | 1 +
 data/NetworkManager-wait-online.service.in                | 1 +
 2 files changed, 2 insertions(+)

diff --git a/data/NetworkManager-wait-online-systemd-pre200.service.in b/data/NetworkManager-wait-online-systemd-pre200.service.in
index 8b54f48..55e4508 100644
--- a/data/NetworkManager-wait-online-systemd-pre200.service.in
+++ b/data/NetworkManager-wait-online-systemd-pre200.service.in
@@ -9,6 +9,7 @@ Before=network.target
 [Service]
 Type=oneshot
 ExecStart=@bindir@/nm-online -s -q --timeout=30
+RemainAfterExit=yes
 
 [Install]
 WantedBy=network.target
diff --git a/data/NetworkManager-wait-online.service.in b/data/NetworkManager-wait-online.service.in
index 4df854c..1753d20 100644
--- a/data/NetworkManager-wait-online.service.in
+++ b/data/NetworkManager-wait-online.service.in
@@ -8,6 +8,7 @@ Before=network-online.target
 [Service]
 Type=oneshot
 ExecStart=@bindir@/nm-online -s -q --timeout=30
+RemainAfterExit=yes
 
 [Install]
 WantedBy=network-online.target
